packages/application-shell/src/components/app-bar/app-bar.js

```diff
@@ -12,79 +12,93 @@ import { REQUESTS_IN_FLIGHT_LOADER_DOM_ID } from '../requests-in-flight-loader/c
 import messages from './messages';
 import styles from './app-bar.mod.css';
 
-const AppBar = props => (
-  <div className={styles['app-bar']} data-test="top-navigation">
-    <Spacings.Inline>
-      <div className={styles.logo}>
-        <img src={LogoSVG} className={styles['logo-img']} alt="Logo" />
-      </div>
+const AppBar = props => {
+  const previouslyUsedProjectKey = selectProjectKeyFromLocalStorage();
+  return (
+    <div className={styles['app-bar']} data-test="top-navigation">
+      <Spacings.Inline>
+        <div className={styles.logo}>
+          {!props.user ? (
+            <LoadingPlaceholder shape="square" size="l" />
+          ) : (
+            <Link
+              to={`/${previouslyUsedProjectKey ||
+                props.user.defaultProjectKey}/dashboard`}
```

Comment:
That's why I need the LoadingPlaceholder now, because we need info from the user which can be loading state 👍

Reply:
Can we not redirect to the dashboard but only the root?